style.css

```diff
@@ -0,0 +1,174 @@
+.navigation-bar{
+    gap:60%;
+}
+
+body{
+    background: rgb(224, 220, 220);
```

Comment:
try using color variables so you can reuse them

Reply:
Okay somesh will follow that thanku